<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManage</class>
 <widget class="QWidget" name="UserManage">
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">

   <!-- 用户信息输入区 -->
   <item>
    <widget class="QGroupBox" name="groupBox_userinfo">
     <property name="title">
      <string>用户信息</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_userinfo">
      <item>
        <widget class="QLabel" name="label_cardid">
          <property name="text">
            <string>卡号：</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="lineEdit_cardid"/>
      </item>
      <item>
        <widget class="QLabel" name="label_name">
          <property name="text">
            <string>姓名：</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item>
        <widget class="QLabel" name="label_gender">
          <property name="text">
            <string>性别：</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="lineEdit_gender"/>
      </item>
      <item>
        <widget class="QLabel" name="label_age">
          <property name="text">
            <string>年龄：</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="lineEdit_age"/>
      </item>
      <item>
        <widget class="QLabel" name="label_tel">
          <property name="text">
            <string>手机号：</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="lineEdit_tel"/>
      </item>
     </layout>
    </widget>
   </item>

   <!-- 上方四个主要按钮 -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_update">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_mainbtns">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>

   <!-- 用户表格区 -->
   <item>
    <widget class="QGroupBox" name="groupBox_usertable">
     <property name="title">
      <string>用户列表</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_table">
      <item>
       <widget class="QTableWidget" name="tableWidget_user"/>
      </item>
     </layout>
    </widget>
   </item>

   <!-- 下方三个功能按钮 -->
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_editbtns">
     <item>
      <widget class="QPushButton" name="pushButton_lost">
       <property name="text">
        <string>挂失</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_find">
       <property name="text">
        <string>取消挂失</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_logout">
       <property name="text">
        <string>注销用户</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_editbtns">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
